sources/HUBComponentWrapper.m

```diff
@@ -118,6 +120,11 @@ - (void)saveComponentUIState
     }
 }
 
+- (nullable HUBComponentWrapper *)childComponentAtIndex:(NSUInteger)index
+{
+    return self.childrenByIndex[@(index)];
```

Comment:
This doesn't only contain visible children. It contains all children created. So the docs doesn't match 🙂